aerofiles/seeyou/reader.py

```diff
@@ -63,7 +63,7 @@ def read(self, fp):
     def decode_waypoint(self, fields):
         # Ignore header line
         if fields == ['name', 'code', 'country', 'lat', 'lon', 'elev',
-                      'style', 'rwdir', 'rwlen', 'freq', 'desc']:
+                      'style', 'rwdir', 'rwlen', 'rwwidth', 'freq', 'desc', 'userdata', 'pics']:
```

Comment:
I think this should be handled similar as in XCSoar: https://github.com/XCSoar/XCSoar/blob/fdd5ffe9013d2dfc4120fd405acddc07ea4a2bec/src/Waypoint/WaypointReaderSeeYou.cpp 

it checks if field number 9 is rwwidth in the line 0 header, and then shifts the other fields. 

My only issue with this is, that the cup file standard documentation, says that the header field is optional. So these "new" fields require to have the header to function.

Reply:
What about making the entire parsing dynamic so that the fields can be any number and in any order. That should make it parse whatever we get, instead of having to use hardcoded field sets. To that a sanity check to make sure that some required fields are present can be done.

Reply:
I wouldn't do that, as in the end we want a valid file that is according to the standard. It wouldn't be good if it passes areofiles but fails in other devices.

Reply:
Well, as I said sanity checks would be done after parsing. The purpose would be to not have hard-coded stuff in the parsing. But it doesn't matter, the current way works fine too, so never mind :)